### DISCUSSION TITLE
Time picker doesn't accept new value when clicking out

### DISCUSSION THREAD

**Discussion by Larsundso** [solspace/craft-freeform#924] [Feature Requests]
**Describe the bug or issue you're experiencing**
When setting a time in the FreeForm Date-Picker using your keyboard and clicking out of the picker without pressing Enter or similar, FreeForm does not accept the inserted value as new time.

**Steps to reproduce**

1. Create form with a Date Picker, time selection being enabled
2. Enter the form
3. Select Date field
4. Select time in Date field
5. Insert time using keyboard
6. Click out of the Date picker, without hitting enter or similar

**Expected behavior**
FreeForm should accept and refresh the value when users don't hit enter and don't use the arrow buttons inside the time picker

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.5.10
- Freeform Version: 4.1.12.1
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh Install

**Additional context**

**Comment by kjmartens** [comment]
Thanks for your feedback, @Larsundso.

This is something that could get fairly complicated to address. However, it's on our list to revisit in the future (as well as possibly adding support for other third-party libraries). 🙂
